deps/key-value-storage/src/postgres/mod.rs

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) 2025 by Alibaba.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! PostgreSQL backend for the key-value storage.
+
+use std::sync::Arc;
+
+use anyhow::{anyhow, Context};
+use async_trait::async_trait;
+use base64::Engine;
+use derivative::Derivative;
+use serde::Deserialize;
+use sqlx::PgPool;
+use sqlx::{postgres::PgPoolOptions, query, Row};
+use tracing::{debug, info, instrument};
+
+use crate::{KeyValueStorage, KeyValueStorageError, Result};
+
+#[derive(Deserialize, Derivative, Clone, PartialEq)]
+#[derivative(Debug)]
+pub struct Config {
+    /// The name of the PostgreSQL database.
+    #[serde(default = "default_db")]
```

Comment:
we can implement `Default` for Config, and then the set `#[serde(default)]` on the whole struct, we don't need all the default_ fn'.